README.md

```diff
@@ -206,6 +206,7 @@ Locate the cell below of the cell with the given label.
 
 Parameters:
 - `label` label to match.
+- `max_row` Optional. Default value is 1. Indicate the number of rows below of label.
```

Comment:
Default value is 1. Indicate the number of rows to search for non empty cell below of label.